src/map/lua/lua_baseentity.h

```diff
@@ -350,7 +350,7 @@ class CLuaBaseEntity
     int32 addHP(lua_State*);                // Modify hp of Entity +/-
     int32 setHP(lua_State*);                // Set hp of Entity to value
     int32 restoreHP(lua_State*);            // Modify hp of Entity, but check if alive first
-    int32 delHP(lua_State*);                // Subtract hp of Entity
```

Comment:
woah woah, there may still be times we need the old binding in a script in the future - don't deleted it even if everything current is using `takeDamage()` after this pr.

Reply:
Added it back in